### ISSUE TITLE
how to apply src_mask in LSTransformerEncoderLayer forward function?

### ISSUE THREAD

**Issue by jz0909** [bytedance/lightseq#341]
I'm trying to use lightseq in my pytorch model training, and I plan to replace torch.nn.TransformerEncoderLayer with LSTransformerEncoderLayer. 

I need to pass src_mask in the pytorch [forward](https://pytorch.org/docs/stable/generated/torch.nn.TransformerEncoderLayer.html#torch.nn.TransformerEncoderLayer.forward) function. Could someone tell me the src_mask equivalent in lighseq?

Thanks

**Comment by Taka152** [comment]
src_mask param is removed in the master branch, and lightseq inference will generate the mask based on padding_id. If you want to pass custom src_mask, you can checkout tag v2.2.0.

**Comment by jz0909** [comment]
Thanks @Taka152 for the reply.
So we cannot use src_mask in future versions of lightseq? 

In the master branch, there is a boolean argument called mask_future_tokens. I think it applies the same change as my src_mask does, do you have any plan to expose it in the encoder layer forward function?
